x-pack/plugins/uptime/public/uptime_app.tsx

```diff
@@ -106,15 +106,17 @@ class Application extends React.Component<UptimeAppProps, UptimeAppState> {
     let colors: UptimeAppColors;
     if (darkMode) {
```

Comment:
@cchaos Do you have a pattern for this kind of thing I could steal. I mostly just recolored what they had here, but it seems like a silly way to do this when the colors vars themselves are the same outside of the theme.

Reply:
I have not run into any theme vars imported via JSON in any of my fixes yet. You could ask SIEM or APM since I'm pretty sure they do. My initial thought would be to set the import path based on the global theme but I'm not exactly sure the way to do this.